src/v1beta1/policy_tag_manager_client.js

```diff
@@ -0,0 +1,982 @@
+      name: request.name,
+    });
+
+    return this._innerApiCalls.deletePolicyTag(request, options, callback);
+  }
+
+  /**
+   * Updates a policy tag.
+   *
+   * @param {Object} request
+   *   The request object that will be sent.
+   * @param {Object} [request.policyTag]
+   *   The policy tag to update. Only the description, display_name, and
+   *   parent_policy_tag fields can be updated.
+   *
+   *   This object should have the same structure as [PolicyTag]{@link google.cloud.datacatalog.v1beta1.PolicyTag}
+   * @param {Object} [request.updateMask]
+   *   The update mask applies to the resource. Only display_name, description and
+   *   parent_policy_tag can be updated and thus can be listed in the mask. If
+   *   update_mask is not provided, all allowed fields (i.e. display_name,
+   *   description and parent) will be updated. For more information including the
+   *   `FieldMask` definition, see
+   *   https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#fieldmask
+   *   If not set, defaults to all of the fields that are allowed to update.
+   *
+   *   This object should have the same structure as [FieldMask]{@link google.protobuf.FieldMask}
+   * @param {Object} [options]
+   *   Optional parameters. You can override the default settings for this call, e.g, timeout,
+   *   retries, paginations, etc. See [gax.CallOptions]{@link https://googleapis.github.io/gax-nodejs/interfaces/CallOptions.html} for the details.
+   * @param {function(?Error, ?Object)} [callback]
+   *   The function which will be called with the result of the API call.
+   *
+   *   The second parameter to the callback is an object representing [PolicyTag]{@link google.cloud.datacatalog.v1beta1.PolicyTag}.
+   * @returns {Promise} - The promise which resolves to an array.
+   *   The first element of the array is an object representing [PolicyTag]{@link google.cloud.datacatalog.v1beta1.PolicyTag}.
+   *   The promise has a method named "cancel" which cancels the ongoing API call.
+   *
+   * @example
+   *
+   * const datacatalog = require('@google-cloud/datacatalog');
+   *
+   * const client = new datacatalog.v1beta1.PolicyTagManagerClient({
+   *   // optional auth parameters.
+   * });
+   *
+   *
+   * client.updatePolicyTag({})
+   *   .then(responses => {
+   *     const response = responses[0];
+   *     // doThingsWith(response)
+   *   })
+   *   .catch(err => {
+   *     console.error(err);
+   *   });
+   */
+  updatePolicyTag(request, options, callback) {
+    if (options instanceof Function && callback === undefined) {
+      callback = options;
+      options = {};
+    }
+    request = request || {};
+    options = options || {};
+    options.otherArgs = options.otherArgs || {};
+    options.otherArgs.headers = options.otherArgs.headers || {};
+    options.otherArgs.headers[
+      'x-goog-request-params'
+    ] = gax.routingHeader.fromParams({
+      'policy_tag.name': request.policyTag.name,
```

Comment:
I see this reads from `policyTag.name`, but it's not showing as a parameter in the function's jsdoc.